server/CVCMIServer.cpp

```diff
@@ -607,11 +608,11 @@ void handleLinuxSignal(int sig)
 }
 #endif
 
-int main(int argc, char** argv)
+int main(int argc, char * * argv)

```

Comment:
Ehh, well sometimes it's nice to have double pointer, but if we have only 13 matches for them, then we can change it manually. In any case we should think about such things :).